### DISCUSSION TITLE
Support for DEX trading

### DISCUSSION THREAD

**Discussion by ch3rrydrunk** [Drakkar-Software/OctoBot#2435] [Ideas]
Hi there! And thank you for an awesome tooling that I really enjoyed working with!

I was thinking about moving out from centralized exchanges to AMMs trading (like UniSwap, Curve etc). Are there any plans towards supporting DEX trading strateegies?

I understand, it would probably require a separate engine or smthng, but I would like to hear your thoughts on the matter and possibly collaborate!

**Comment by Kristia3** [comment]
Hi! The idea of ​​moving from centralized exchanges to AMM trading sounds great. I would recommend Uniswap
